### PR TITLE
[ONE] Implement All Will Be One

### PR DESCRIPTION
**Pull request by PurpleCrowbar**

### CODE REVIEW

**Comment by PurpleCrowbar** [comment]
[[All Will Be One]]

File: Mage.Sets/src/mage/cards/a/AllWillBeOne.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    }
+
+    @Override
+    public boolean checkEventType(GameEvent event, Game game) {
+        return event.getType() == GameEvent.EventType.COUNTERS_ADDED;
+    }
+
+    @Override
+    public boolean checkTrigger(GameEvent event, Game game) {
+        if (event.getPlayerId() != getControllerId()) {
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/a/AllWillBeOne.java:77]: you should use `isControlledBy` here

File: Mage.Sets/src/mage/cards/a/AllWillBeOne.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        super(card);
+    }
+
+    @Override
+    public AllWillBeOne copy() {
+        return new AllWillBeOne(this);
+    }
+}
+
+class AllWillBeOneFilter extends FilterCreaturePlayerOrPlaneswalker {
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/a/AllWillBeOne.java:41]: this doesn't need to be a separate class, just make a static `FilterCreaturePlayerOrPlaneswalker` in the triggered ability class and modify it

**Review by theelk801 — CHANGES_REQUESTED** [review]

File: Mage.Sets/src/mage/cards/a/AllWillBeOne.java (hunk 1 above)
Comment (PurpleCrowbar) [reply Mage.Sets/src/mage/cards/a/AllWillBeOne.java]: Done

File: Mage.Sets/src/mage/cards/a/AllWillBeOne.java (hunk 2 above)
Comment (PurpleCrowbar) [reply Mage.Sets/src/mage/cards/a/AllWillBeOne.java]: Done

**Pull request merged**